src/YQMenuBar.cc

```diff
@@ -238,3 +238,13 @@ YQMenuBar::activateItem( YMenuItem * item )
     if ( item )
         YQUI::ui()->sendEvent( new YMenuEvent( item ) );
 }
+
+
+void
+YQMenuBar::shortcutChanged()
+{
+    // Any of the items might have its keyboard shortcut changed, but we don't
+    // know which one. So let's simply rebuild the menu bar again.
+
+    rebuildMenuTree();
```

Comment:
Hm... if multiple shortcuts were changed, this is probably called multiple times. Rebuilding the menu tree is quite an expensive operation.

Let's keep it like this for now, but please add a FIXME comment here so this isn't completely forgotten; we'll need to address this some time in the future.